sig-and-wg/sig-core/proposals/objects-store-proposal.md

```diff
@@ -0,0 +1,155 @@
+# Objects Store
```

Comment:
I suggest to rename it to `ObjectStore Orchestrator`

Reply:
I'm not sure we can really call it orchestrator, but maybe it is just me thinking that orchestrator is something big :)

maybe you could elaborate more

Reply:
through direct feedback I also get suggestion to remove `Object` from the name as it is to generic and also kind of hard to position and explain with Object Storage name in general.

Suggested name is AssetStore and looking on use cases I think I ❤️ it

Reply:
will wait with renaming though as I'm interested what others mean